src/main/kotlin/org/rust/cargo/toolchain/ProxyHelper.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.toolchain
+
+import com.intellij.util.net.HttpConfigurable
+import java.net.URI
+
+class ProxyHelper(var httpSettings: HttpConfigurable) {
```

Comment:
Field could be private. Also, why do we need it as a constructor parameter? Perhaps it would be simpler to create a zero-argument constructor and call `HttpConfigurable.getInstance()` inside it?